### PR TITLE
Toast container

### PR DESCRIPTION
**Pull request by eddielegg-le**
### Summary
Change implementation of toast messages to support business date mismatch message:
- Changed library for messages to support multiple messages
  - Required multiple changes, including style changes and implementation of custom toast component
- Updated Toast class to accept an icon
- Added `info` color scheme

### Screenshots
Samples of toast messages
image

Business Date Message
image

### CODE REVIEW

**Review by blubaugha — APPROVED** [review]

File: openpos-flow/src/main/java/org/jumpmind/pos/core/ui/Toast.java [hunk 1]
```diff
@@ -11,6 +13,10 @@
     private ToastType toastType;
     private int duration = 2500;
     private String verticalPosition = "bottom";
+    private String icon;
+    private boolean persistent;
+    private UUID persistedId;
+    private boolean close;
```
Comment (eddielegg-le) [inline openpos-flow/src/main/java/org/jumpmind/pos/core/ui/Toast.java:19]: Think maybe this should be a different message? Maybe a `CloseToast` class with just the ID?

**Comment by mmichalek** [comment]
Eddie this is looking good.  Any chance we can widen that toast, so the business date one is one line and allows the logo to peek through the back.
image

**Review by mmichalek — COMMENTED** [review]
See widen comment.

**Comment by eddielegg-le** [comment]
> Eddie this is looking good. Any chance we can widen that toast, so the business date one is one line and allows the logo to peek through the back.

Sure can:

image

**Review by blubaugha — APPROVED** [review]

**Review by mmichalek — APPROVED** [review]

**Pull request merged**